docs/wrap-with-providers.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { ThemeProvider } from "@kiwicom/orbit-components";
+
+import { DevModeProvider } from "./src/hooks/useDevMode";
+import { TableOfContentsProvider } from "./src/components/TableOfContents";
+import theme from "./src/theme";
+
+export default function wrapWithProviders({ element }: { element: React.ReactNode }) {
+  return (
+    <ThemeProvider theme={theme}>
+      <DevModeProvider>
+        <TableOfContentsProvider>{element}</TableOfContentsProvider>
+      </DevModeProvider>
+    </ThemeProvider>
```

Comment:
The main reason why I moved these providers here is because I needed the context for table of contents to already be available in `DocLayout` in order to check whether there are any items, so I decided to do some extra optimization and moved other repeating providers here as well, as suggested by Viktor. Now we won't get into problems with missing context and having to repetitively provide it.